base/src/org/eevolution/model/MPPOrder.java

```diff
@@ -845,11 +839,11 @@ public boolean closeIt()
 		
 		for(MPPOrderBOMLine line : getLines())
 		{
-			BigDecimal old = line.getQtyRequired();
-			if (old.compareTo(line.getQtyDelivered()) != 0)
+			BigDecimal qtyRequired = line.getQtyRequired();
+			if (qtyRequired.compareTo(line.getQtyDelivered()) != 0)
 			{	
 				line.setQtyRequired(line.getQtyDelivered());
-				line.addDescription(Msg.parseTranslation(getCtx(), "@closed@ @QtyRequired@ (" + old + ")"));
+				line.addDescription(Msg.parseTranslation(getCtx(), "@closed@ @QtyRequired@ (" + qtyRequired + ")"));

```

Comment:
Please add a better definition for this, toy can use 
```Java
line.addDescription(Msg.parseTranslation(getCtx(), "@closed@ @QtyRequired@ (" + DisplayType.getNumberFormat(DisplayType.Quantity).format(qtyRequired) + ")"));
```

Reply:
@yamelsenih Hi Yamel!

This code is not new, it is a legacy code, it was only refactored and separated into methods to make the code more readable.

I can modify the legacy code, but this makes it more difficult to follow the changes, I will modify what is possible thus avoiding introducing any problem.

Reply:
I modify , but the mendable not is very different that https://github.com/adempiere/adempiere/blob/develop/base/src/org/compiere/model/MOrder.java#L2071